### ISSUE TITLE
Add Fofix

### ISSUE THREAD

**Issue by kestouf** [recalbox/recalbox-os#1163]
Hello
This is not a bug, but I did not find or make a proposal to add. Sorry.
I would like to know if it was possible to add FOFIX to your recalbox. FOfix is frets on fire running Linux. Frets on fire is an open source Guitar Hero.

**Comment by digitalLumberjack** [comment]
Yeah god idea. I know fof runs on python, so we need to validate that it's not too heavy for rpi. We will add that to ports.

**Comment by kestouf** [comment]
If onRIP is not possible, it's  possible to add easyest on X86 buile? (I use an Intel NUC)

**Comment by kestouf** [comment]
I the too fofix is OK on ODROID C2.
